### ISSUE TITLE
Is https://popmotion.io/api/tween/ is completely out of date?

### ISSUE THREAD

**Issue by mbylstra** [Popmotion/popmotion#898]
As far as I can tell there is no longer a function named `tween`, as is reported by my typescript compiler, and from looking at the source code. I am very confused. Has this been replaced with a function simply named `animate`? Why do these docs still exist? If they are there for people on an old versoin, there should be a big warning sign saying that they're out of date and pointing people at the latest version. I can't remember how I ended up at that url, perhaps from a google search.

**Comment by mattgperry** [comment]
Yeah good point, we'll add a deprecation warning at the top of these pages.

**Comment by mbylstra** [comment]
Thanks! This is a fantastic library by the way - exactly what I was looking for!

**Comment by JoshuaKGoldberg** [comment]
Ping @mattgperry, this holds up for at least `chain` and `delay` too. The upgrade guide is missing content: https://popmotion.io/blog/20190704-upgrade-to-pure-9/

Is there any guidance for how to bring in the great changes from version 9, please? 🌹

**Comment by mattrossman** [comment]
Please please add that deprecation notice. I just wasted quite a bit of time trying to learn how to use popmotion from https://popmotion.io/learn and couldn't figure out why I couldn't import `value`.
